### PR TITLE
Implemented resolver returning proposal & proposal list 

### LINKED ISSUE #15: Index Governance proposals and voting data for frontend
Reference to client issue

https://github.com/forbole/big-dipper-celo-client/issues/20

### PR DESCRIPTION
**Pull request by MonikaCat**
Fixes #15 

Implemented `Proposal` and `Proposals` GraphQL schema and resolvers that to query list of proposals or individual proposal details

### CODE REVIEW

File: server/main.ts [hunk 1]
```diff
@@ -127,6 +140,7 @@ Meteor.startup(() => {
         updateBlock(number);
         updateTokenPrice();
         updatePendingTransactions();
+        getAllProposals();
```
Comment (kwunyeung) [inline server/main.ts:149]: Does it mean it will only be called once at startup?

**Review by kwunyeung — COMMENTED** [review]
Can be merged if we confirm the proposals are being tracked continuously.

**Review by kwunyeung — APPROVED** [review]
LGTM

**Comment by kwunyeung** [comment]
@MonikaCat should this close #15 ?

**Comment by MonikaCat** [comment]
> @MonikaCat should this close #15 ?

Yes, it will close #15. I have updated the PR description :)

File: server/main.ts (hunk 1 above)
Comment (MonikaCat) [reply server/main.ts:149]: Right! I have added a timer to `getAllProposals()` method to call every 5 blocks

**Pull request merged**